demo/capture_config.yaml

```diff
@@ -0,0 +1,140 @@
+lcr-modules:
+
+    _shared:
+        lcr-modules: "../"
+        lcr-scripts: "../../lcr-scripts/"
+        root_output_dir: "results/"
+        scratch_directory: "scratch/"
+        unmatched_normal_ids:
+            capture--grch37: "TCRBOA7-N-WEX"
+    
+    battenberg:
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {sample_id}
+            sample_bam: "data/{sample_id}.bam"
+        
+        reference_path:
+            hg38: " --chr_prefixed_genome "                           
+            grch37: " reference/genomes/grch37/downloads/hg19-grch37 "  
+        
+    mutect2:
+        inputs:
+            sample_bam: "data/{sample_id}.bam"
+            sample_bai: "data/{sample_id}.bam.bai"
+    
+    varscan:
+        inputs:
+            sample_bam: "data/{sample_id}.bam"
+            sample_bai: "data/{sample_id}.bam.bai"
+
+    liftover:
+        tool: "sequenza-1.4" 
+        dirs: 
+            _parent: "results/sequenza-1.4_liftover-1.2"
+        inputs:
+            sample_seg: "results/sequenza-1.4/03-igv_seg/genome--grch37/TCRBOA7-T-WGS--TCRBOA7-N-WGS--matched/filtered/sequenza_segments.igv.seg"
```

Comment:
can you please point it here to `99-outputs` instead? If in the future a new step will be added to sequenza workflow, this might break. Instead, 99-outputs are consistently meant to have the final results from each module